charts/ingress-nginx/ci/daemonset-extra-modules.yaml

```diff
@@ -0,0 +1,10 @@
+controller:
+  kind: DaemonSet
+  image:
+    repository: ingress-controller/controller
+    tag: 1.0.0-dev
+  service:
+    type: ClusterIP
+  extraModules:
+    - name: opentelemetry
+      image: busybox
```

Comment:
once we merge this, we should probably change the CI and point to the right module :)

Reply:
Yes, absolutely.